src/main/java/org/spongepowered/common/data/processor/data/entity/GlowingDataProcessor.java

```diff
@@ -0,0 +1,58 @@
+package org.spongepowered.common.data.processor.data.entity;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import net.minecraft.entity.Entity;
+import org.spongepowered.api.data.DataTransactionResult;
+import org.spongepowered.api.data.key.Keys;
+import org.spongepowered.api.data.manipulator.immutable.entity.ImmutableGlowingData;
+import org.spongepowered.api.data.manipulator.mutable.entity.GlowingData;
+import org.spongepowered.api.data.value.ValueContainer;
+import org.spongepowered.api.data.value.immutable.ImmutableValue;
+import org.spongepowered.api.data.value.mutable.Value;
+import org.spongepowered.api.util.OptBool;
+import org.spongepowered.common.data.manipulator.mutable.entity.SpongeGlowingData;
+import org.spongepowered.common.data.processor.common.AbstractEntitySingleDataProcessor;
+import org.spongepowered.common.data.util.DataConstants;
+import org.spongepowered.common.data.value.immutable.ImmutableSpongeValue;
+import org.spongepowered.common.data.value.mutable.SpongeValue;
+
+import java.util.Optional;
+
+public class GlowingDataProcessor extends AbstractEntitySingleDataProcessor<Entity, Boolean, Value<Boolean>, GlowingData, ImmutableGlowingData> {
+
+    public GlowingDataProcessor() {
+        super(Entity.class, Keys.GLOWING);
+    }
+
+    @Override
+    protected boolean set(Entity dataHolder, Boolean value) {
+        dataHolder.setGlowing(checkNotNull(value, "value"));
+        return true;
+    }
+
+    @Override
+    protected Optional<Boolean> getVal(Entity dataHolder) {
+        return OptBool.of(dataHolder.isGlowing());
```

Comment:
My eyes appear to have played a trick on me when I commented on that. :smile_cat: